lib/takeaway.rb

```diff
@@ -0,0 +1,32 @@
+require './lib/menu'
+require './lib/message'
+class Takeaway
+  attr_reader :order, :total
+  def initialize(menu_klass=Menu.new)
+      @menu = menu_klass
+      @order = []
+      @total = 0
+  end
+
+  def list_dishes
+    @menu.dishes.map{ |dish| "#{dish[:name]}: £#{dish[:price]}"}*", "
+  end
+
+  def add(item, quantity = 1)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for add is too high. [16.31/15] (http://c2.com/cgi/wiki?AbcMetric)